### DISCUSSION TITLE
taxonomy term - template is not shown - why?

### DISCUSSION THREAD

**Discussion by torbentschechne** [statamic/cms#4711] [Troubleshooting] [answered]
Hello,
I don't see how I can access a template of a term and the content of the term's entry. I have a collection `faq` and a taxonomy `tags`. I connected the taxonomy `tags` to the collection `faq`. The route for this collection is `faq/{slug}`. I have two tags, `newsletter` and `ecommerce` - these tags are selected in the entries of the collection `faq`. 

Now I list all entries which are tagged with `newsletter` on a `tags/newsletter` page and each item on that page is linked to `faq/{slug}` but somehow I can not see the contents of the term, when I click on the link.

So for example in the collection "faq" I have an entry with the slug "my-first-entry". And I go to `domain.com/faq/my-first-entry` non of these templates are called:

<img width="310" alt="CleanShot 2021-11-11 at 21 54 22@2x" src="https://user-images.githubusercontent.com/4470136/141367580-fab1941a-a398-4667-a415-a4adf6fa6003.png">

What am I missing here? Currently what happens is that some items of the pages are included twice, when I visit the url `faq/my-first-entry`:

![CleanShot 2021-11-11 at 21 55 22@2x](https://user-images.githubusercontent.com/4470136/141367771-df30a869-d5fc-4b80-b52e-e5d3ada87514.png)

Thanks!

**Comment by edalzell** [comment, marked as answer]
Are you able to create a repo with this issue so we can take a look?

**Reply by torbentschechne** [reply]
Hey @edalzell,

thanks for getting back to me. I created a repo here: https://github.com/torbentschechne/statamic-test

I discovered another issue. When you visit the FAQ section, you'll see two items "newsletter" and "ecommerce". Once you click on each of them, you see that both pages show the same entries, even when not all entries are tagged with both terms.

I did it like this:
```
{{ collection:faq taxonomy:tags="{ tags|option_list }"}}
```
So I assume that there might be something very broken with my taxonomies...maybe there are still v2 related things which must be completely different in v3.

Thank you very much that you take a look. I hope you'll send me a link where I can donate a beer or so to you :)

**Reply by torbentschechne** [reply]
@edalzell I think I figured it out. I went through the Statamic tutorial on Youtube regarding taxonomies.

So I identified that after I created an index and show template in `views/faq/tags/` I could visit the URL `domain.com/faq/tags` and `domain.com/faq/tags/newsletter`. That worked.

Then I saw that my FAQ page which should list those tags is doing that:

```
{{ taxonomy:tags }}
            <div class="w-1/3 px-2">
                <a class="block text-center border-2 border-gray-600 rounded pt-8 pb-8" href="{{ url }}"><strong>{{ title }}</strong></a>
            </div>
{{ /taxonomy:tags }}
```

And somehow the url's here were not `domain.com/faq/tags/newsletter`, but `domain.com/tags/newsletter`. This did not work - and here I do not understand why the global snippet `{{ taxonomy:tags }}` can not pull the tags, when they are in `faq/tags` ?

So I moved the the `tags` folder from `views/faq` to `views` to match the url structure and that's it now. It seems to work now.
